### DISCUSSION TITLE
Crankshaft uses same port

### DISCUSSION THREAD

**Discussion by konqiDAM** [SteamDeckHomebrew/decky-loader#144] [Q&A] [answered]
Hello there, I think Crankshaft uses same port, I tried to use both at the same time and had issues until I disabled Crankshaft. Maybe you can talk with the other dev and try not to collide. Or maybe was just a random coincidence I had. This is the error I was getting:

![image](https://user-images.githubusercontent.com/44301655/185505556-50424fbc-25a6-4210-8923-949300f086eb.png)

**Comment by TrainDoctor** [comment, marked as answer]
decky-loader does as much as is feasibly possibly to not collide other software that modifies the deck's game mode UI but because crankshaft just straight up edits the DOM and JS of the Deck's UI it will always collide with decky-loader. We suggest you take the issue up with the developer of crankshaft.

**Comment by gatlinnewhouse** [comment]
went ahead and opened an issue with crankshaft: https://github.com/CrankshaftForSteam/crankshaft/issues/13
